tests/test_http_ext_auth.py

```diff
@@ -187,30 +187,55 @@ def http_con_send_request(self, *args, headers=None, **kwargs):
             headers['x-edgedb-oauth-test-server'] = test_port
         return super().http_con_send_request(*args, headers=headers, **kwargs)
 
+    async def get_client_config_by_provider(self, provider_name: str):
+        return await self.con.query_single(
+            """
+            SELECT assert_exists(assert_single(
+                cfg::Config.extensions[is ext::auth::AuthConfig]
+                    .providers { * } filter .provider_name = <str>$0
+            ));
+            """,
+            provider_name,
+        )
+
+    async def get_auth_config_value(self, key: str):
+        return await self.con.query_single(
+            f"""
+            SELECT assert_single(
+                cfg::Config.extensions[is ext::auth::AuthConfig]
+                    .{key}
```

Comment:
Is there a way to parameterize this in the query rather than interpolating the value into the query string itself?

Reply:
No as that would make the result type undecidable.  All queries must type check at compile time.